vtest/gen-ref.bat

```diff
@@ -1,7 +1,7 @@
 
 rem create reference
 
-set MSCORE=..\msvc.install_x64\bin\musescore.exe
```

Comment:
@igorkorsukov take a look on that

Reply:
oops

Reply:
Was wrong before, and still ids ;-)

Reply:
It is a much simpler script and not meant to be run automatically, and so %INSTALL_PATH% is not set, so I guess we can let this pass for now?

Reply:
I think yes, we can pass this 
This is files not used on CI, as I understand, they for local testing on Windows?

Reply:
Yes